packages/@core.next/auth.jsx

```diff
@@ -139,7 +139,7 @@ const AuthProvider = ({ children, initialUserValue }) => {
 
 if (DEBUG_RERENDERS_BY_WHY_DID_YOU_RENDER) AuthProvider.whyDidYouRender = true
 
-const initOnRestore = async (ctx) => {
+export const initOnRestore = async (ctx) => {
```

Comment:
This variable is not a part of a public API. You should pass an arguments for `useAuth`